packages/opentelemetry-plugin-fetch/src/fetch.ts

```diff
@@ -63,10 +64,13 @@ export class FetchPlugin extends core.BasePlugin<Promise<Response>> {
     span: api.Span,
     corsPreFlightRequest: PerformanceResourceTiming
   ): void {
-    const childSpan = this._tracer.startSpan('CORS Preflight', {
-      parent: span,
-      startTime: corsPreFlightRequest[web.PerformanceTimingNames.FETCH_START],
-    });
+    const childSpan = this._tracer.startSpan(
+      'CORS Preflight',
+      {
+        startTime: corsPreFlightRequest[web.PerformanceTimingNames.FETCH_START],
+      },
+      setActiveSpan(api.context.active(), span)
```

Comment:
the same `api.context.active()` would be enough here

Reply:
Previously it was setting parent explicitly. I did this to maintain the same behavior. I am not as familiar with the fetch plugin though.